build.sbt

```diff
@@ -1,73 +1,22 @@
 import sbt.Path.relativeTo
 
-ThisBuild / majorVersion := 1
-ThisBuild / isPublicArtefact := true
-
-val scala2_12 = "2.12.18"
 val scala2_13 = "2.13.12"
 
-def copySources(module: Project) = Seq(
-  Compile / scalaSource := (module / Compile / scalaSource).value,
-  Compile / resourceDirectory := (module / Compile / resourceDirectory).value,
-  Test / scalaSource := (module / Test / scalaSource).value,
-  Test / resourceDirectory := (module / Test / resourceDirectory).value
-)
-
-lazy val library = (project in file("."))
-  .settings(publish / skip := true)
-  .aggregate(
-    sys.env.get("PLAY_VERSION") match {
-      case Some("2.8") => playNunjucksPlay28
-      case Some("2.9") => playNunjucksPlay29
-      case _           => playNunjucksPlay30
-    }
-  )
+ThisBuild / majorVersion := 1
+ThisBuild / isPublicArtefact := true
+ThisBuild / scalaVersion := scala2_13
 
 lazy val playNunjucksPlay30 = Project("play-nunjucks-play-30", file("play-nunjucks-play-30"))
   .enablePlugins(SbtWeb)
-  .settings(inConfig(Test)(testSettings): _*)
-  .settings(scalaVersion := scala2_13)
   .settings(
     libraryDependencies ++= LibDependencies.play30,
     coverageExcludedPackages := "<empty>;uk.gov.hmrc.BuildInfo;uk.gov.hmrc.nunjucks.PlayModuleRegistry",
-    buildInfoKeys ++= Seq[BuildInfoKey]("playVersion" -> LibDependencies.play30Version)
+    buildInfoKeys ++= Seq[BuildInfoKey]("playVersion" -> LibDependencies.play30Version),
+    Compile / resourceGenerators += npmModulesTarballTask.taskValue,
+    JsEngineKeys.engineType := JsEngineKeys.EngineType.Node,
+    JsEngineKeys.npmPreferSystemInstalledNpm := true,
```

Comment:
think it's the above two are the default now, but just to be explicit, these are just used during build when we download node modules - won't impact services, they will still be using the trireme? dependency to run nunjucks js in jvm